examples/fundamentals.test.js

```diff
@@ -4418,6 +4418,52 @@ describe('Fundamentals', function() {
       arr.findIndex(n => Number.isNaN(n)); // 4
       
     });
+    it('bad-undefined-check', function() {
+      var person = { firstName:"John", lastName:"Doe", age:50, eyeColor:"blue", test: undefined };
+      // acquit:ignore:start
+      assert.throws(() => {
+      // acquit:ignore:end
+
+      if ( x == undefined ) {
+        console.log('will throw a reference error');
```

Comment:
I'd just make this example `x === undefined; // Throws a ReferenceError" rather than adding an `if` statement